### PR TITLE
add_evaluation_delay of datadog monitor

### PR DESCRIPTION
**Pull request by usaturn**
##### SUMMARY

- Supported new_host_delay of DataDog Monitor

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

- datadog_monitor

##### ANSIBLE VERSION

```
ansible 2.4.1.0
```

### CODE REVIEW

**Comment by skornehl** [comment]
Thanks @usaturn!

shipit

**Comment by usaturn** [comment]
When will this be merged?
Thank you.

**Comment by heavenshell** [comment]
Plz shipit

**Comment by tsuyukimakoto** [comment]
shipit :-)

**Comment by usaturn** [comment]
@skornehl

Hi!
Please remove the `community_review` label and ship it!

**Comment by usaturn** [comment]
@mkrizek 
Please respond to this issue.

File: lib/ansible/modules/monitoring/datadog_monitor.py [hunk 1]
```diff
@@ -117,6 +117,11 @@
         required: false
         default: null
         version_added: "2.4"
+    evaluation_delay:
+        description: ["Time to delay evaluation (in seconds). It is effective for sparse values."]
+        required: false
+        default: null
+        version_added: "2.5"
```
Comment (mkrizek) [inline lib/ansible/modules/monitoring/datadog_monitor.py]: Please change this to `2.7`.

**Comment by usaturn** [comment]
@mkrizek
updated. Thank you.

**Pull request merged**